asctec_hl_gps/package.xml

```diff
@@ -21,6 +21,7 @@
   <build_depend>message_filters</build_depend>
   <build_depend>std_srvs</build_depend>
   <build_depend>cmake_modules</build_depend>
+  <build_depend>message_generation</build_depend>
```

Comment:
is this really necessary? I'd assume std_srvs should bring it in. But see next comment.